src/Raven.Studio/typescript/commands/licensing/forceLicenseUpdateCommand.ts

```diff
@@ -1,16 +1,30 @@
 import commandBase = require("commands/commandBase");
 import endpoints = require("endpoints");
+import LicenseLeaseResult = Raven.Server.Commercial.LicenseLeaseResult;
+import genUtils = require("common/generalUtils");
 
 class forceLicenseUpdateCommand extends commandBase {
-
-    execute(): JQueryPromise<boolean> {
+    execute(): JQueryPromise<LicenseLeaseResult> {
         const url = endpoints.global.license.adminLicenseForceUpdate;
-        return this.post(url, null, null)
-            .done(() => this.reportSuccess("Your license was successfully updated"))
+
+        return this.post<LicenseLeaseResult>(url, null, null)
+            .done((result: LicenseLeaseResult) => {
+                switch (result.Status) {
+                    case "NotModified":
+                        this.reportWarning("Your license has not been modified");
```

Comment:
why? 
Since there is no need to do this? Can we rework that to be more informative?

Reply:
"The license was fetched successfully, but is still expired"?

Reply:
Note that we can call the `Force Update` also for a non-expired license.
If it's the same license, we'll get the same response.

Reply:
You can check if it's expired from the license status.

Reply:
So if endopint returns "NotModified", and licenseStatus.Expired is:
	- true -> "The license was fetched successfully, but is the same"
	- false -> "The license was fetched successfully, but is still expired"